webapi/src/main/scala/org/knora/webapi/Main.scala

```diff
@@ -30,6 +31,10 @@ object Main extends ZIOApp {
   ] = ZLayer.empty ++ Runtime.removeDefaultLoggers ++ SLF4J.slf4j ++ LayersLive.dspLayersLive
 
   /* Here we start our Application */
-  override def run = AppServer.live.launch
-
+  override def run = for {
+    f1 <- PrometheusServer.make.forkDaemon
+    f2 <- (AppServer.live *> ZIO.never).forkDaemon
+    _  <- f1.join
+    _  <- f2.join
```

Comment:
```suggestion
    _ <- PrometheusServer.make.forkDaemon.join
    _ <- (AppServer.live *> ZIO.never).forkDaemon.join
```

How about join in the same lines?